gsoc/gsoc2016/ideas_yt.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: default
+title:  "Ideas for yt"
+show_main: false
+ideas_team: yt
+---
+
+### Integrate yt plots with interactive matplotlib backends
+
+*Suggested Mentor(s):* Nathan Goldbaum
+
+*Difficulty:* Intermediate
+
+*Knowledge needed:* Familiarity with matplotlib. Knowledge of matplotlib internals a plus
+
+*Programming skills:* Python. GUI programming.
+
+#### Description
+
+Currently, all yt plotting objects have a `show()` method that currently
+displays a version of the plot in Jupyter notebooks. This works for the most
+part and is relatively simple due to Jupyter's data model. However, this
+reliance on the notebook fails for users who work primarily from the command
+line, either in the vanilla python interpreter or the IPython command line
+application. We receive many requests from confused users who do not understand
+why `show()` errors out in the regular python interpreter or appears to do
+nothing in `IPython`, when they expect a GUI window to pop up when they run
+`show()`.
+
+This project would have a student modify yt's plotting objects to hook into
+matplotlib's interactive backends so that plots can be optionally displayed
+using a GUI plotting window. Optimally, we would also enable callbacks so that
+zooming and selecting does the "right thing", generating high resolution data
+when it is available.
+
+This is constrained by maintaining backward compatibility: by default yt should
+not fail when generating plots on headless devices (e.g. when connecting over
+SSH to a supercomputer).
+
+
+
+### Testing improvements
+
+*Suggested Mentor(s)"* Kacper Kowalik, Nathan Goldbaum
```

Comment:
the `"` should be a `:`.
Also, it would be good if the names are linked to their profiles (either in GH or BB).